bin/update-modules-translations.mjs

```diff
@@ -0,0 +1,51 @@
+/* eslint-env node */
+/**
+ * This script is used for downloading the latest Zendesk official translation files for the modules in the `src/module` folder.
+ *
+ */
+import { writeFile } from "node:fs/promises";
+import { resolve } from "node:path";
+
+/**
+ *  Maps each folder in the `src/modules` directory with its package name on the translation system
+ */
+const MODULES = {
+  "new-request-form": "new-request-form",
+};
```

Comment:
I was thinking if we could eventually find a way to have this work in a "convention over configuration" type of way where it wouldn't be needed to manually keep this map. Maybe by collocating the translation within each app and check which module contains those file.
Also totally fine to keep it this way as we can always further improve.

Reply:
> Got it! One can argue that it is not a traditional fetch call and the file should actually exist and be loaded properly for the app to work as expected.
Where I am getting at is that this might actually be one of the errors that we do want.
>
> Totally fine to keep this behaviour though 👍🏼

We don't have the guarantee that a file for a `base_locale` exists at runtime. For example, at the moment we have a `tl-ph` locale that doesn't have a `parent_identifier`, but we don't have translations for that locale.

It is probably an error in the JSON containing the mapping, but even if it gets fixed it can happen again in the future when new locales are added, or if for some reason we don't have a translation file for a locale available.

If we don't handle the error, the app will crash immediately. I think it is better to handle the error and let the app use the English strings as a fallback.

> I was thinking if we could eventually find a way to have this work in a "convention over configuration" type of way where it wouldn't be needed to manually keep this map. Maybe by collocating the translation within each app and check which module contains those file.
Also totally fine to keep it this way as we can always further improve.

It is a good point. Actually, we already have the package name in the translations YAML file. It is possible to change the script to read the package name from there, instead of manually keeping the map.
Let's merge this PR and then we can eventually revisit it